### PR TITLE
Fix scss and postcss

### PR DESCRIPTION
**Pull request by karlhorky**
~~**Blocked** on #11~~

### CODE REVIEW

File: packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js [hunk 1]
```diff
@@ -32,17 +32,18 @@ const AppMode = archetype.AppMode;
...
+const sassExists = (glob.sync(Path.resolve(AppMode.src.client, "**", "*.scss")).length > 0);
 
 // By default, this archetype assumes you are using CSS-Modules + CSS-Next
-const cssModuleSupport = stylusExists && !cssNextExists;
+const cssModuleSupport = sassExists && !cssNextExists;
 
 module.exports = function () {
-  const cssModuleStylusSupport = archetype.webpack.cssModuleStylusSupport;
-  const stylusQuery = cssLoader + "?-autoprefixer!" + stylusLoader;
+  // TODO: Can we get rid of these unused variables?
```
Comment (karlhorky) [inline packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js]: There are 2 TODOs here that should be looked at as well before merging.

File: packages/generator-electrode-component/app/index.js [hunk 2]
```diff
@@ -141,7 +141,7 @@ var ReactComponentGenerator = yeoman.generators.Base.extend({
     },
     demo: function () {
       this.template("demo/_demo.jsx", "demo/demo.jsx");
-      this.template("demo/_demo.css", "demo/demo.css");
+      this.template("demo/_demo.css", "demo/demo.scss");
       this.template("demo/examples/_component.example", "demo/examples/" + this.projectName + ".example");
```
Comment (kaytcat) [inline packages/generator-electrode-component/app/index.js]: should demo/_demo.css also be scss?

File: packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js [hunk 3]
```diff
@@ -19,16 +20,14 @@ const postcssLoader = require.resolve("postcss-loader");
...
 /**
- * [cssModuleSupport By default, this archetype assumes you are using CSS-Modules + CSS-Next]
+ * [cssModuleSupport By default, this archetype assumes you are using SCSS + CSS-Modules + CSS-Next]
  *
  * Stylus is also supported for which the following cases can occur.
  *
  * case 1: *only* *.css exists => CSS-Modules + CSS-Next
- * case 2: *only* *.styl exists => stylus
- * case 3: *both* *.css & *.styl exists => CSS-Modules + CSS-Next takes priority
+ * case 2: *only* *.scss exists => SCSS + CSS-Modules + CSS-Next
```
Comment (kaytcat) [inline packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js]: Is CSS-Next here correct?

File: packages/electrode-archetype-react-component-dev/README.md [hunk 4]
```diff
@@ -45,10 +45,10 @@ gulp, webpack, demo, electrode component archetype, live-reload
     create package.json
     create README.md
     create src/components/product-card.jsx
-    create src/styles/product-card.styl
+    create src/styles/product-card.scss
```
Comment (karlhorky) [inline packages/electrode-archetype-react-component-dev/README.md]: Not entirely sure if we need to edit the component archetypes. We don't publish these to npm yet (not prefixed with `kununu-`).

cc @FreaKzero

File: packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js (hunk 3 above)
Comment (dennisat) [inline packages/kununu-electrode-archetype-react-app-dev/config/webpack/partial/extract-style.js]: yes

File: packages/electrode-archetype-react-component-dev/README.md [hunk 5]
```diff
@@ -142,14 +142,14 @@ package.json
 ## CSS Modules
 
 By default, this archetype assumes you are using CSS-Modules + CSS-Next, you need
-to opt-in to stylus support by adding a `*.styl` to your *project's* `demo/demo.styl` & `src/styles/*.styl`
+to opt-in to stylus support by adding a `*.scss` to your *project's* `demo/demo.scss` & `src/styles/*.scss`
```
Comment (dennisat) [inline packages/electrode-archetype-react-component-dev/README.md]: only Stylus

File: packages/generator-electrode-component/app/index.js (hunk 2 above)
Comment (dennisat) [inline packages/generator-electrode-component/app/index.js]: we can have css and/or scss (everywhere)

File: packages/electrode-archetype-react-component-dev/README.md (hunk 4 above)
Comment (dennisat) [inline packages/electrode-archetype-react-component-dev/README.md]: I am going to fix this

**Pull request merged**